etna/transforms/base.py

```diff
@@ -161,6 +165,18 @@ def inverse_transform(self, ts: TSDataset) -> TSDataset:
         """
         pass
 
+    def params_to_tune(self) -> Dict[str, "BaseDistribution"]:
+        """Get grid for tuning hyperparameters.
+
+        This is default implementation with empty grid.
+
+        Returns
+        -------
+        :
+            Empty grid.
+        """
+        return {}
```

Comment:
I feel like we should make it abstract just not forget to implement it

Reply:
I think that empty default implementation is reasonable enough. Users doesn't need to think about it if they don't need it.